operate/qa/integration-tests/pom.xml

```diff
@@ -43,6 +43,12 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>io.camunda</groupId>
+      <artifactId>operate-importer-8_7</artifactId>
```

Comment:
AFAIK the 8.7 importers will be removed since the new CamundaExporter will be taking over for 8.7. Why is this and the respective `tasklist-importer` required?

Reply:
I'm also confused, it's not related to my changes but I just added it to make the pipleline green while checking for dependency:analayze